adding_species_manuscript.tex

```diff
@@ -153,18 +153,18 @@
 
 \section*{Abstract}
 
-Simulation is a key tool in population genetics for both methods development and empirical research,
+Simulation is a key tool in population genetics for both method development and empirical research,
 but producing simulations that recapitulate the main features of genomic datasets remains a major obstacle.
```

Comment:
I think that methods development is actually the more common term to use in this context